### ISSUE TITLE
[New build - EXPERIMENTAL] 21w02a: Unity Version Upgrade

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#3464]
The version of the Unity game engine has been updated.
To allow this the netcode solution had to be migrated too.
Please report any regression!

Bugfixes:
#3314 Remote Player invisible inside ship in multiplayer.

**Comment by tsunamayo** [comment]
Build is live! This is a technical one, tell me if you find anything.

**Comment by Womble-TC** [comment]
Awesome! Anything specific you want us to look out for in terms of MP performance?

**Comment by tsunamayo** [comment]
Thanks! I would say first check if the issue of missing ship on load is better or not...

**Comment by tsunamayo** [comment]
I have a bug where I cant spawn two ship. The first one works, the second starter block is bugged out.

**Comment by Womble-TC** [comment]
I have the same bug, I can spawn multiple ships as long as I choose a BP on the first starter block. However, it seems 2 starter blocks cannot exist at once.

**Comment by tsunamayo** [comment]
yep this is a bit annoying, I already fix that issue, I will fix a couple more and push a new build.
